<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password_modify_func</class>
 <widget class="QWidget" name="Password_modify_func">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>241</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>319</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>319</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
border-image: url(:/background/屏幕截图 2024-06-01 172200.png);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgba(255, 255, 255,0.3);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 12pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px dashed #fffac2;
	border-radius: 3px;
	background-color:#fffee2;
	color: #000000;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
QLineEdit:focus {
    border: 1px dashed #fffac2;
    border-radius: 3px;
    background-color: #fffee2;
    color: #000000;
    selection-background-color: #298DFF;
    selection-color: #F2F2F2;
}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入新密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>40</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgba(255, 255, 255,0.3);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 12pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px dashed #fffac2;
	border-radius: 3px;
	background-color:#fffee2;
	color: #000000;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
QLineEdit:focus {
    border: 1px dashed #fffac2;
    border-radius: 3px;
    background-color: #fffee2;
    color: #000000;
    selection-background-color: #298DFF;
    selection-color: #F2F2F2;
}

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgba(255, 255, 255,0.3);/* 透明度 */
border-radius: 3px; /* 边框圆角 */
font-size: 12pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 1px dashed #fffac2;
	border-radius: 3px;
	background-color:#fffee2;
	color: #000000;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
QLineEdit:focus {
    border: 1px dashed #fffac2;
    border-radius: 3px;
    background-color: #fffee2;
    color: #000000;
    selection-background-color: #298DFF;
    selection-color: #F2F2F2;
}

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.6);</string>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
